js/index.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-unused-vars */
+const mobileMenuModulesCreator = (() => {
+  // eslint-disable-next-line quotes
```

Comment:
- [ ] It would be better to remove the `Eslint disable ` because Eslint ensures that you write clean and better code so please make sure to do that